data/grants_comparison/mesh_tree_letters_list.txt

```diff
@@ -0,0 +1,4 @@
+Information Sources: L
```

Comment:
so what's this?

Reply:
This is for the evaluation. It's the result of [this discussion](https://wellcome.slack.com/archives/C02JM7DC3UP/p1687169778045799) with Kirstin. 
In the MeSH terms we were provided, we also got some higher level codes included i.e. non-leaf nodes as I understand. I've stored those inside this file.